packages/common/src/Widget.ts

```diff
@@ -584,9 +584,11 @@ export abstract class Widget extends PropertyExt {
         });
     }
 
-    lazyRender = debounce(function () {
-        this.render();
-    }, 100);
+    lazyRender = function (_debouncedCallback) {
```

Comment:
underbars in params should only be used to indicate params are not referenced in a function (optionally).